lib/models/addon.js

```diff
@@ -1061,18 +1063,6 @@ let addonProto = {
       this.options.babel = this.__originalOptions.babel;
     }
 
-    let emberCLIBabelConfigKey = this._emberCLIBabelConfigKey();
-    if (!this.options[emberCLIBabelConfigKey] || !this.options[emberCLIBabelConfigKey].compileModules) {
```

Comment:
Lets bring this back, and change it around a bit to reset the internal `this.options` state to match what we setup in the constructor.

Reply:
done